app/slashcommands/command_invite.go

```diff
@@ -137,17 +137,25 @@ func (i *InviteProvider) parseMessage(a *app.App, c request.CTX, args *model.Com
 	return targetUsers, targetChannels, ""
 }
 
-func (i *InviteProvider) getUserProfile(a *app.App, username string) *model.User {
-	userProfile, nErr := a.Srv().Store().User().GetByUsername(username)
-	if nErr != nil {
-		return nil
+func (i *InviteProvider) getUsersFromMention(a *app.App, username string) []*model.User {
+	users := make([]*model.User, 0, 1)
+
+	userProfile, err := a.Srv().Store().User().GetByUsername(username)
+	if err == nil && userProfile.DeleteAt == 0 {
+		users = append(users, userProfile)
+	}
```

Comment:
@kostaspt - Wouldn't we want to return early here if it's in fact a user, instead of attempting to also look at it as a group?

Reply:
Good catch, thanks!